kani-compiler/src/codegen_cprover_gotoc/codegen/span.rs

```diff
@@ -53,6 +53,14 @@ impl<'tcx> GotocCtx<'tcx> {
         self.current_fn().mir().var_debug_info.iter().find(|info| match info.value {
             VarDebugInfoContents::Place(p) => p.local == *l && p.projection.len() == 0,
             VarDebugInfoContents::Const(_) => false,
+            // This variant was added in
+            // https://github.com/rust-lang/rust/pull/102570 and is concerned
+            // with a scalar replacement of aggregates (SROA) MIR optimization
+            // that is only enabled with `--mir-opt-level=3` or higher.
+            // TODO: create a test and figure out if we should return debug info
+            // for this case:
+            // https://github.com/model-checking/kani/issues/1933
+            VarDebugInfoContents::Composite { .. } => false,
```

Comment:
Can you please add a `debug_assert!(false)` for now?

Reply:
Is a `todo` or `unreachable` OK?

Reply:
I think what we discussed is that returning `false` here should be OK, since this only affects the variable name that gets generated.

Reply:
Oh, I see that you're proposing a `debug_assert` so that this only fails internally. I'll add that.